ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Ledger/NetworkProtocolVersion.hs

```diff
@@ -19,6 +19,9 @@ data ByronNodeToNodeVersion =
 
     -- | We send headers /with/ a size hint
   | ByronNodeToNodeVersion2
+
+    -- | KeepAlive protocol
+  | ByronNodeToNodeVersion3
```

Comment:
This is not needed, in #2358 we decoupled binary serialisation format changes from protocol changes.